pkg/services/secrets/kvstore/helpers.go

```diff
@@ -27,3 +28,54 @@ func SetupTestService(t *testing.T) SecretsKVStore {
 
 	return kv
 }
+
+// In memory kv store used for testing
+type FakeSecretsKVStore struct {
```

Comment:
Can it be secrets/kvstore/fakes ? The problem is that we are using kvstore.Key so it results in a cycle import.

Reply:
Ah, I see what you mean. Based on how everything is structured, it doesn't look like it makes sense to move it in there. 

Maybe we can make a /test directory under kvstore and add all of our test files + the helper to there

Reply:
Proposed:
![image](https://user-images.githubusercontent.com/34773040/179307051-5a010942-1c65-496a-b67f-0ac7401424e2.png)
The problem with this is that `secretsKVStoreSQL` is private, and is needed by helpers.go

Without doing a big refactor, the only way I can make it work is leave everyting (tests, helpers, fake) under kvstore folder.

Reply:
Like this:
![image](https://user-images.githubusercontent.com/34773040/179307509-55dc6309-cbae-4238-a2d9-0f41ea803b23.png)

The thing is that to hide the implementations (which is a pro, as it can be confusing) is difficult to mock and test them

Reply:
I think we should refrain from big refactors out of the scope of the PR, but it would be good to add a draft task to our board to restructure tests on secrets kvstore in the future on a different PR.

Reply:
Yeah, seems like way too much trouble for now. Definitely fine as-is.

Reply:
Agree, created this task https://github.com/grafana/grafana-partnerships-team/issues/276